app/cluster/dynamic.go

```diff
@@ -164,6 +165,9 @@ func (d *Dynamic) start() error {
 	if err := d.RouterDB.Start(); err != nil {
 		return fmt.Errorf("router db start: %w", err)
 	}
+	if err := d.EventSchemasDB.Start(); err != nil {
+		return fmt.Errorf("event schemas db start: %w", err)
+	}
 	if err := d.BatchRouterDB.Start(); err != nil {
 		return fmt.Errorf("batch router db start: %w", err)
```

Comment:
```
func (d *Dynamic) start() (err error) {
	if d.GatewayComponent {
		return nil
	}
	d.logger.Info("Starting the server")

	defer func() {
		if r := recover(); r != nil {
			err = r.(error)
		}
	}()
	start := time.Now()
	lo.Must0(d.ErrorDB.Start(), "error db start")
	lo.Must0(d.GatewayDB.Start(), "gateway db start")
	lo.Must0(d.EventSchemaDB.Start(), "event schemas db start")
	lo.Must0(d.RouterDB.Start(), "router db start")
	lo.Must0(d.BatchRouterDB.Start(), "batch router db start")
	lo.Must0(d.MultiTenantStat.Start(), "multi tenant stat start")
	lo.Must0(d.Processor.Start(), "processor start")
	lo.Must0(d.Router.Start(), "router start")
	d.serverStartTimeStat.SendTiming(time.Since(start))
	d.serverStartCountStat.Increment()
	return
}
```
Not to suggest this change, but would something like this behave the exact same way as the current start function..?

Reply:
What if something else panics, other than `lo.Must0`? Error handling in Go is better to be performed by returning errors, instead of recovering from panics.

Reply:
Understood.
so if there were only several `lo.Must0(...start())` in the `*Dynamic.start()` and nothing else that could panic, such a pattern would behave the same way..?